### PR TITLE
Add tcp addon limitations

### LINKED ISSUE #200: [TCP] Add IP limitation
A user won't be able to find the real IP since the SNAT is changing it

### PR DESCRIPTION
**Pull request by johnsudaar**
Fixes #200

### CODE REVIEW

File: _posts/addons/2017-10-23-tcp-addon.md [hunk 1]
```diff
@@ -69,6 +69,20 @@ communication must pass via our load balancer. It's the exact same thing if you
 need to contact a `web` container from a `tcp` container, you must use the
 public URL of the `web` container.
 
+### Application Restart and New Deployments
```
Comment (yannski) [inline _posts/addons/2017-10-23-tcp-addon.md]: no mention of the SYN handshake test?

File: _posts/addons/2017-10-23-tcp-addon.md [hunk 2]
```diff
@@ -96,6 +110,13 @@ A limit burst is granted for every container. This burst is equal to 10 times
 the limit, so if you have a limit of 10 packets/s, your burst will be set to 100
 packets.
 
+## Limitations
+
+To be able to load balance tcp connections, connections are proxied to your
```
Comment (yannski) [inline _posts/addons/2017-10-23-tcp-addon.md]: -> "Thus, the device identification should be part of the protocole you designed over the TCP connection"?

**Comment by yannski** [comment]
BUMP @johnsudaar lots of unanswered questions here

**Comment by johnsudaar** [comment]
Better @yannski ?

**Pull request merged**